yarn.lock

```diff
@@ -2261,9 +2436,11 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/catalog-model@^0.2.0":
-  version "0.6.0"
+  version "0.2.0"
```

Comment:
these @backstage changes here and below seem accidental and should be removed

Reply:
They appear frequently when doing `yarn add` or `yarn remove`! Yet to create that issue for more discussion on this problem/hack for Roadie plugins.

@ayshiff You could manually revert these `@backstage` changes. I sometimes copy over the `yarn.lock` file from the master branch and running a fresh `yarn`.